up42/estimation.py

```diff
@@ -29,7 +29,7 @@ def __init__(
                 },]
         """
         self.auth = auth
-        self.project_id = project_id
+        self.project_id = project_id if project_id is not None else self.auth.credentials_id
```

Comment:
I know this can be the opposite of your idea to add the project_id but if this is not included it would mean a breaking change on the client side. Please, correct me if I am wrong.